Code/IO/include/sitkImageReaderBase.h

```diff
@@ -72,12 +72,28 @@ class SmartPointer;
        * Unknown private tags may be encoded with Base64 encoding.
        * @{
        */
-      virtual Self& SetLoadPrivateTags(bool loadPrivateTags);
+      virtual SITK_RETURN_SELF_TYPE_HEADER  SetLoadPrivateTags(bool loadPrivateTags);
       virtual bool GetLoadPrivateTags() const;
       virtual void LoadPrivateTagsOn();
       virtual void LoadPrivateTagsOff();
       /* @} */
 
+
+      /** \brief Set/Get name of ImageIO to use
+       *
+       * An option to override the automatically detected ImageIO used
+       * to read the image. The available ImageIOs are listed by the
+       * GetRegisteredImageIOs method. If the ImageIO can not be
+       * constructed an exception will be generated. If the ImageIO
+       * can not read the file an exception will be generated.
+       *
+       * By the value the value is and empty string "". This default
+       * value indicates that the proper ImageIO will be automatically determined.
+       */
```

Comment:
"By the value..." , sentence doesn't make sense to me. Should this read:
The default value is the empty string (""). This indicates that the ImageIO will be determined automatically.